eradiate/scenes/tests/test_measure.py

```diff
@@ -112,6 +112,9 @@ def test_perspective(mode_mono):
     d = PerspectiveCameraMeasure()
     assert KernelDict.empty().add(d).load() is not None
 
+    with pytest.raises(AttributeError):
+        d = PerspectiveCameraMeasure(origin=[0,0,0], target=[0,0,0])
```

Comment:
Please try with a value other that [0, 0, 0] (this test could not detect the mistakes you made).